### PR TITLE
Add Zebrium tile

### PR DESCRIPTION
**Pull request by rodbagg**
### What does this PR do?

Adds Zebrium tile in integrations-extra 

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by alexvetras** [comment]
Hi @rodbagg, to display a video in your media gallery, please send our team the zipped file or a link to download the video at marketplace@datadog.com. 

Once approved, we will upload the file to Vimeo and it will be displayed in the media gallery. Please note that the gallery can only hold one video. We will provide the Vimeo ID to use as a comment in your PR.

In the meantime, can you please remove the mp4 file you submitted with this pull-request?

File: .github/CODEOWNERS
Comment (anawishnoff) [inline .github/CODEOWNERS:127]: Looks like this is throwing an error - did you purposefully remove this email from the codeowners file?

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:7]:
```suggestion
Using Zebrium is simple. When you're troubleshooting a problem, instead of digging around, simply add the Zebrium app to your Datadog dashboard and look at the details of the corresponding detection.
```

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:9]:
```suggestion
Zebrium integrates with Datadog in two ways: 1) a Datadog App with a custom dashboard widget and 2) an events and metrics integration.
```

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:11]:
```suggestion
### 1) Datadog App
```

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:13]:
```suggestion
The Zebrium app provides a pre-built, elegant and interactive dashboard widget to see the Root Cause of any problems (as well as a few relevant metrics). This method provides the simplest user experience.
```

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:38]:
```suggestion
   - **Token**: Paste the token that you created in step 6 above.
```

File: zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json
Comment (anawishnoff) [inline zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json:13]: I'd recommend making this title more descriptive about what data the widget is showing

File: zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json
Comment (anawishnoff) [inline zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json:1]: For partner-built integrations, we recommend that integration dashboards also include your logo as well as a description of the data being presented.

File: zebrium/assets/service_checks.json
Comment (anawishnoff) [inline zebrium/assets/service_checks.json:1]: This file can be deleted.

File: zebrium/images/IMAGES_README.md
Comment (anawishnoff) [inline zebrium/images/IMAGES_README.md:1]: This file can be deleted.

File: zebrium/manifest.json
Comment (anawishnoff) [inline zebrium/manifest.json:19]: I'd recommend making this a bit more descriptive as to what Zebrium does specifically

File: .github/CODEOWNERS
Comment (rodbagg) [reply .github/CODEOWNERS:127]: I added that email addr back in

File: zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json
Comment (rodbagg) [reply zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json:13]: We would like to leave this one as-is since it's branding for the Zebrium product

File: zebrium/assets/service_checks.json
Comment (rodbagg) [reply zebrium/assets/service_checks.json:1]: Done

File: zebrium/images/IMAGES_README.md
Comment (rodbagg) [reply zebrium/images/IMAGES_README.md:1]: done

File: zebrium/manifest.json
Comment (rodbagg) [reply zebrium/manifest.json:19]: Done

File: zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json
Comment (rodbagg) [reply zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json:1]: We'll adda logo and desc.

**Comment by rodbagg** [comment]
I have committed all changes from the review on 4/20

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:5]:
```suggestion
When you know there's a problem and you're not sure what caused it, [Zebrium][1] shows you the root cause directly on your Datadog dashboards. It works by using machine learning on logs, without the need for manual training or setting up any rules, and it achieves accuracy in less than 24 hours. 
```

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:7]:
```suggestion
Using Zebrium is simple. When troubleshooting a problem, simply add the Zebrium app to your Datadog dashboard and look at the details of the corresponding detection.
```

File: zebrium/README.md
Comment (anawishnoff) [inline zebrium/README.md:13]:
```suggestion
The Zebrium app provides a pre-built, elegant and interactive dashboard widget that displays the detected problems over time and allows you to dig into the Root Cause of these problems (as well as a few relevant metrics). This method provides the simplest user experience.
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
A couple more comments. Thanks!

File: zebrium/README.md
Comment (cswatt) [inline zebrium/README.md:7]:
```suggestion
When troubleshooting a problem, add the Zebrium app to your Datadog dashboard and look at the details of the corresponding detection.
```

File: zebrium/README.md
Comment (cswatt) [inline zebrium/README.md:13]:
```suggestion
The Zebrium app provides a pre-built, interactive dashboard widget that displays the root cause of any problems (as well as a few relevant metrics). This method provides the simplest user experience.
```

File: zebrium/README.md
Comment (cswatt) [inline zebrium/README.md:52]:
```suggestion
Need help? Contact [Datadog Support][4].


[1]: https://www.zebrium.com
[2]: https://app.datadoghq.com/organization-settings/api-keys
[3]: https://docs.zebrium.com/docs/monitoring/datadog_autodetect/
[4]: http://docs.datadoghq.com/help
[5]: https://cloud.zebrium.com
```

**Review by cswatt — CHANGES_REQUESTED** [review]
docs style review

File: zebrium/manifest.json
Comment (rodbagg) [reply zebrium/manifest.json]: Done
https://github.com/DataDog/integrations-extras/pull/1276/commits/f47bd2e6412dde8e4279a8071e1bd9caa2d8ccfb

File: zebrium/manifest.json
Comment (fanny-jiang) [inline zebrium/manifest.json:58]:
```suggestion
      },
      "service_checks": {}
```
Manifest validation is failing on a missing mandatory `assets[service_checks]` field.

**Review by fanny-jiang — CHANGES_REQUESTED** [review]

**Comment by anawishnoff** [comment]
@rodbagg The app has been approved and good to go, I believe the only outstanding issue here is @fanny-jiang's comment above. Thanks!

File: zebrium/manifest.json
Comment (rodbagg) [reply zebrium/manifest.json]: @anawishnoff - I deleted the service_checks section and it failed validation. I added it back but failed again because metadata_path is a required field. Added that with an empty string and that too failed. I am not sure what I need to do to proceed

Comment (anawishnoff) [reply zebrium/manifest.json]: @yzhan289 Any insight here as to how to correctly define that there's no service checks?

**Review by cswatt — APPROVED** [review]

File: zebrium/manifest.json
Comment (yzhan289) [inline zebrium/manifest.json:59]: For validations to pass, you will need to create an empty `service_checks.json` like so: https://github.com/Samrose-Ahmed/integrations-extras/blob/01af012a79f9af52285eda16a75e04ab7395626b/apptrail/assets/service_checks.json

Comment (rodbagg) [reply zebrium/manifest.json:59]: thx. Fixed

**Review by sarah-witt — CHANGES_REQUESTED** [review]
Hi @rodbagg can you please add a `CHANGELOG.md` file in the `zebrium/` directory as mentioned in the [`manifest.json`](https://github.com/DataDog/integrations-extras/pull/1276/files#diff-739d921a95ee2c5d1d456005d53283c8909d6b1b0936f813c1932b0dc7d457e9R18)?

**Comment by rodbagg** [comment]
@sarah-witt - I have added CHANGELOG.md

**Review by sarah-witt — APPROVED** [review]
Looks good to me!

**Comment by anawishnoff** [comment]
Hi @rodbagg, we've synced this tile to our staging environment and found just a couple last things to tie up:
- The title gets covered by the "Install" button on the tile (screenshot below). Would you be able to shorten the `title` in the `manifest.json` file? Perhaps just "Zebrium" would be best?
    - Screen Shot 2022-05-11 at 4 39 06 PM 
    - I also noticed your logo is off-center, which is an issue on our end and we will fix.
- I believe there were some UI updates made, and [this screenshot](https://github.com/rodbagg/integrations-extras/blob/master/zebrium/images/Zebrium_Root_Cause_Finder_With_Side_Panel.png) is out of date. Would you be able to update that screenshot?

**Comment by rodbagg** [comment]
@anawishnoff - I have shorted the title and updated the widget images. Thanks!

**Comment by rodbagg** [comment]
Done
    On Wednesday, May 11, 2022, 01:48:26 PM PDT, Ana Wishnoff ***@***.***> wrote:  
 
 


Hi @rodbagg, we've synced this tile to our staging environment and found just a couple last things to tie up:
   
   - The title gets covered by the "Install" button on the tile (screenshot below). Would you be able to shorten the title in the manifest.json file? Perhaps just "Zebrium" would be best?      
      - 
      - I also noticed your logo is off-center, which is an issue on our end and we will fix.

   - I believe there were some UI updates made, and this screenshot is out of date. Would you be able to update that screenshot?

File: zebrium/README.md
Comment (alexvetras) [inline zebrium/README.md:13]:
```suggestion
The Zebrium app provides a pre-built, interactive dashboard widget that displays the detected problems over time, and allows you to dig into the Root Cause of these problems (as well as relevant metrics). This method provides the simplest user experience.
```

**Review by sarah-witt — APPROVED** [review]

**Pull request merged**
